lib/inferno/entities/test_group.rb

```diff
@@ -80,8 +80,8 @@ def reference_hash
           }
         end
 
-        def run_as_group
-          @run_as_group = true
+        def run_as_group(value: true)
```

Comment:
I don't think we want a keyword argument here. `value = true`

Reply:
I tried that first, but rubocop said to "use keyword arguments when defining method with boolean argument"

Reply:
That doesn't make sense in this case, so I think you should disable that rule on this line.